tests/http/v1/governance_test.go

```diff
@@ -37,25 +38,25 @@ func makeTestProposals() []storage.Proposal {
 			ID:           1,
 			Submitter:    "oasis1qpydpeyjrneq20kh2jz2809lew6d9p64yymutlee",
 			State:        "passed",
-			Deposit:      10000000000000,
+			Deposit:      *big.NewInt(10000000000000),
```

Comment:
Ditto, let's have something >2^64 here. And then in the actual test you can verify that it's returned as a JSON string.